petstore/frontend/react/src/config.js

```diff
@@ -0,0 +1,5 @@
+const config = {
```

Comment:
I moved this file to the top and removed the config directory since this was the only file in it.